graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/QueryFetcherTest.java

```diff
@@ -94,20 +95,42 @@ public void mutationDescriptionTest(String name, String description) {
   }
 
   public static Arguments[] operationsWithOptions() {
+    Parameters defaults = CassandraFetcher.defaultParameters;
     return new Arguments[] {
       arguments(
-          "query { books(options: { pageSize: 100, pageState: \"AWEA8H////4A\", consistency: LOCAL_QUORUM }) { values { title, author } } }",
+          "query { books(options: { pageSize: 101, pageState: \"AWEA8H////4A\", consistency: LOCAL_QUORUM }) { values { title, author } } }",
           ImmutableParameters.builder()
-              .pageSize(100)
+              .from(defaults)
+              .pageSize(101)
               .pagingState(ByteBuffer.wrap(Base64.getDecoder().decode("AWEA8H////4A")))
               .consistencyLevel(ConsistencyLevel.LOCAL_QUORUM)
               .build()),
       arguments(
           "mutation { insertBooks(value: {title:\"a\", author:\"b\"}, options: { consistency: LOCAL_ONE, serialConsistency: SERIAL}) { applied } }",
           ImmutableParameters.builder()
+              .from(defaults)
               .consistencyLevel(ConsistencyLevel.LOCAL_ONE)
               .serialConsistencyLevel(ConsistencyLevel.SERIAL)
-              .build())
+              .build()),
+      arguments(
+          "query { books { values { title, author } } }",
+          ImmutableParameters.builder()
+              .pageSize(100)
+              .consistencyLevel(ConsistencyLevel.LOCAL_QUORUM)
+              .serialConsistencyLevel(ConsistencyLevel.SERIAL)
+              .build()),
```

Comment:
Nit: maybe use the constant here too
```suggestion
          defaults),
```

Reply:
I wanted to explicitly test for those settings, I'll add a comment.